debian/rules

```diff
@@ -31,6 +31,21 @@ override_dh_auto_install-arch:
 	$(BAZEL) run $(BAZEL_BUILD_OPTIONS) //:install -- \
 		--strip_tool=$(STRIP) \
 		/opt/drake
+	@# As of drake 0.39.0, VTK 8 and VTK 9 are installed, with VTK 8 being
+	@# needed exclusively for drake-visualizer.  Internally, drake modifies
+	@# the build tree LD_LIBRARY_PATH and PYTHONPATH to make VTK 8 available
+	@# for drake-visualizer.  The installed drake-visualizer python wrapper
+	@# handles this, however dh_makeshlibs will fail because the VTK 8
+	@# libraries (C++ and Python) have missing rpath entries not able to be
+	@# resolved.  The following two lines patch the rpath entries for VTK 8
+	@# C++ and Python libraries so the dh_makeshlibs checks can pass.  The
+	@# installed ${prefix}/bin/.drake-visualizer-real executable already has
+	@# its rpath set to $ORIGIN/../lib so no patching is necessary.
+	@# Delete these once VTK 8 is no longer installed.
+	find $(DESTDIR)/opt/drake/lib -type f -name "libvtk*-8.2.so.1" \
+		-printf '  patchelf on %p\n' -exec patchelf --set-rpath '$$ORIGIN' {} \;
+	find $(DESTDIR)/opt/drake/lib/python*/site-packages/vtkmodules -name "*.so" \
+		-printf '  patchelf on %p\n' -exec patchelf --set-rpath '$$ORIGIN/../../..' {} \;
```

Comment:
All of the other commands are using `$(DESTDIR)/opt/drake`, however it is not clear to me if this will succeed since `$(BAZEL) run $(BAZEL_BUILD_OPTIONS) //:install -- --strip_tool=$(STRIP) /opt/drake` is run, rather than `$(DESTDIR)/opt/drake`.  The `DESTDIR` is set at the top of the file.  As such, I added some `-printf` in there for the logs, if no entries saying `patchelf on XYZ.so` are printed, then we need to do `/opt/drake` rather than `$(DESTDIR)/opt/drake`.

Or possibly update the bazel install command to do `$(DESTDIR)/opt/drake` rather than `/opt/drake`.